noxfile.py

```diff
@@ -111,12 +111,10 @@ def docs(session):
     session.install("Sphinx < 2.0dev")
     session.install(".")
     # Building the docs.
-    run_args = [
-        "bash",
-        "test_utils/test_utils/scripts/update_docs.sh",
-    ]
+    run_args = ["bash", "test_utils/test_utils/scripts/update_docs.sh"]
```

Comment:
I think blacken did this. Maybe related to lint failure @crwilcox was complaining about yesterday.